scripts/rollup/build.js

```diff
@@ -112,7 +121,13 @@ function getBanner(bundleType, hasteName, filename) {
   }
 }
 
-function getFooter(bundleType) {
+function getFooter(bundleType, filename, moduleType) {
+  if (moduleType === RECONCILER) {
+    return (
+      '\nreturn ReactFiberReconciler(config);\n};\n' +
```

Comment:
This feels wrong. We're relying on a variable name here that can change at any time.

Reply:
(Not that I have any solution here.)